### ISSUE TITLE
Trying to POST to /user gives me 403

### ISSUE THREAD

**Issue by luisincrespo** [tjwebb/sails-auth#51]
Hey there.

I've configured sails-auth as you explain in your readme.

You say that one would normally POST to /user without a problem. But it's giving me 403:

    {
        "error": "You are not permitted to perform this action."
    }

Should I modify the config/policies.js you specify in your readme?.

**Comment by tjwebb** [comment]
> Should I modify the config/policies.js you specify in your readme?.

I'm not sure what this means; are you saying that you knowingly didn't follow all the instructions in the readme and also that something doesn't work?

**Comment by tjwebb** [comment]
@luisincrespo is there a reason you closed this?

**Comment by luisincrespo** [comment]
Hey @tjwebb. Yep I followed all the instructions in the README and was getting the error I stated in the question. However, I closed this because I started using **waterlock**. You referred to it somewhere in your issues, when talking about JWT, which is what I needed. It's working fine for now, thanks for that referral.

**Comment by JHereu** [comment]
I'd like to re-open this issue. I have the same problem. In my case, I just created a new fresh Sails 0.11 app and followed the instructions on how to install the sails-auth module (I'm interested in using local passport login). So, I'm supposed to be able to post users to /user even if I'm not logged, is that right? 'Cause if I have no users stored in my database, I cannot login anywhere, it's kind of obvious.

POST and GET to the /user route gives me the same message as @luisincrespo:

    {
      "error": "You are not permitted to perform this action."
    }

I've already modified the policies as README dictates, and so on.
